first_hour_on_vwb/vwb_org_group_utils.py

```diff
@@ -0,0 +1,41 @@
+import json
+import subprocess
+import sys
+
+def get_org_linked_group_roles(org_id, group_name):
+    """
+    Return a flattened mapping of users to roles for a named org-linked group and org ID.
+    """
+    roles_dict = {role: set() for role in ["ADMIN", "MEMBER", "READER", "SUPPORT"]}
+    
+    wb_command = ["wb","group","role","list",f"--org={org_id}",f"--name={group_name}","--format=JSON"]
+    result = subprocess.run(wb_command,capture_output=True,text=True)
+    nested_roles = json.loads(result.stdout)
+
+    for item in nested_roles:
+        if item['principal']['principalType'] == "GROUP":
+            roles_dict[role].update(get_org_linked_group_roles(item['principal']['groupOrg'], item['principal']['groupName'])["MEMBER"])   
+        else:
+            for role in item['roles']:
+                if item['principal']['userEmail'] != None:
+                    roles_dict[role].add(item['principal']['userEmail'])
+
+    return roles_dict
+
+def get_flat_roles_html(roles_dict):
+    html = f"""<table style='margin: 0 auto,text-align: left'>"""
+    html += "<th>ROLE</th>"
+    html += "<th>USER_EMAILS</th>"
+    html += "<tr>"
```

Comment:
whenever I have to do something like this (construct html from strings) I wonder if there's a better way...

not asking for a change -- I think this is perfectly fine in small doses. just musing that it might be worth some thought someday.